Cargo.toml

```diff
@@ -17,7 +17,7 @@
 
 [package]
 name = "parquet-format"
-version = "2.6.0"
+version = "3.0.0"
```

Comment:
Why is this version 3.0.0? We keep the version in sync with parquet-format version.

Reply:
I think this is because of the backward incompatible changes from Thrift 0.12.0. I think we should stick to 2.6.X if we can avoid that. Otherwise, we may have to bump up the major version.

Reply:
Indeed it's because of the backwards incompatible changes introduced by Thrift 0.12.0. According to [the changelog](https://github.com/apache/thrift/tree/master/lib/rs#thrift-0130) this seems to be non-optional.